README.md

```diff
@@ -60,6 +60,7 @@ export interface DepGraph {
     version: string | null;
   }>>;
   toJSON(): DepGraphData;
+  isSame(other: DepGraph): boolean;
```

Comment:
I'd consider a more standard name for this method: `equals()` is what's commonly used in most ecosystems in my experience. Just a thought

Reply:
It's not `equals`. Because I'm not comparing root node.

Reply:
interesting

Reply:
We can also consider to use `equals` with flag `includingRoot = false`.

Reply:
I'm curious as to the motivation for excluding the root from comparison. I'm sure there's a good reason, I just can't think of it off the top of my head

Reply:
This function initially designed for commit status checks. Just to compare two branches and check if we need to run tests against it.

If it's a library, root node version will change after each release. But we don't care about it because dependencies are same.

Reply:
@Kirill89 nice! didn't about the this. I think it's a good idea to add this `compareRoot: boolean` flag as a param. because in the case of coparing snapshots for example we would like to compare root as swell